.vscode/settings.json

```diff
@@ -2,8 +2,7 @@
   "deno.config": "./deno.jsonc",
   "deno.enable": true,
   "deno.enablePaths": [
-    // everything except .vuepress
-    "./site/README.md",
```

Comment:
Why this exception?

Reply:
No idea. It's been there since the beginning

Reply:
> Why this exception?

Because .vuepress/ contains vuepress plugins and configurations? (Node)

Reply:
I mean why do you no longer want to enable Deno for the file `./site/README.md`? So far, it has worked well to have it enabled.

Reply:
The file was written twice by mistake. Have a look at it again, the README file is still there.